games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="3D open-source go-kart racing game"
+DESCRIPTION="3D open-source go-kart racing game with an extensive variety of characters, tracks, and modes to play."
+HOMEPAGE="https://supertuxkart.net"
+COPYRIGHT="2020 SuperTuxKart Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/supertuxkart/stk-code/releases/download/$portVersion/supertuxkart-$portVersion-src.tar.xz"
+CHECKSUM_SHA256="052edf0afdbeb99583fe8676fb0ab80ecb6103fb88b7540f858d1b5fa1297d37"
+SOURCE_FILENAME="supertuxkart-$portVersion-src.tar.xz"
+SOURCE_DIR="SuperTuxKart-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+ supertuxkart$secondaryArchSuffix = $portVersion
+ app:SuperTuxKart$secondaryArchSuffix
+ lib:libangelscript$secondaryArchSuffix
```

Comment:
angelscript is provided by an external lib: https://repology.org/project/angelscript/versions
Mabye try to see if we could port this to Haiku?
Same for libsquish: https://repology.org/project/libsquish/versions